grails-app/conf/Config.groovy

```diff
@@ -66,10 +66,10 @@ grails.exceptionresolver.params.exclude = ['password']
 grails.converters.json.circular.reference.behaviour = "INSERT_NULL"
 
 // Mail plugin config
-grails.mail.adminEmailAddress = "aatams_admin@emii.org.au"
-grails.mail.systemEmailAddress = "aatams_system@emii.org.au"
-grails.mail.supportEmailAddress = "info@emii.org.au"
-grails.mail.dataCommitteeEmailAddress = "aatams_data_committee@emii.org.au"
+grails.mail.adminEmailAddress = "aatams_admin@aodn.org.au"
+grails.mail.systemEmailAddress = "aatams_system@aodn.org.au"
+grails.mail.supportEmailAddress = "info@aodn.org.au"
+grails.mail.dataCommitteeEmailAddress = "aatams_data_committee@aodn.org.au"
```

Comment:
@kereid Would it be easier if they were all changed to point at just info@aodn.org.au? 

We would lose a bit of context about the purpose of the email, but would avoid having to setup and monitor multiple email accounts.